### PR TITLE
[4.0] css and js form validation

### PR DESCRIPTION
**Pull request by Khanjan04**
Pull Request for Issue #32682 .

### Summary of Changes

Changes done in 3 files- validate.es6.js, blocks_form.scss and bootstrap_form.scss 

### Actual result BEFORE applying this Pull Request

has-success and has-danger

### Expected result AFTER applying this Pull Request

is-valid and is-invalid

### CODE REVIEW

File: templates/system/build_incomplete.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
...
+	<script>window.errorLocale={"en-US":{"language":"English US","header":"Environment Setup Incomplete","text1":"It looks like you are trying to run Joomla! from our git repository. To do so requires you to complete a couple of extra steps first.","help-url-text":"More Details"},"en-GB":{"language":"English GB","header":"Environment Setup Incomplete","text1":"It looks like you are trying to run Joomla! from our git repository. To do so requires you to complete a couple of extra steps first.","help-url-text":"More Details"}};</script><!-- Sets the content of the translated text, IMPORTANT DO NOT REMOVE -->
```
Comment (richard67) [inline templates/system/build_incomplete.html]: This change seems to be unrelated and not intentional since it is not mentioned in the description of this PR. Please revert it.

File: templates/system/fatal-error.html [hunk 2]
```diff
@@ -6,7 +6,7 @@
...
-	<script>window.errorLocale={"en-GB":{"language":"English GB","header":"Sorry, there was a problem we could not recover from.","text1":"The server returned a \"{{statusCode_statusText}}\"","help-url-text":"Help me resolve this"},"en-US":{"language":"English US","header":"Sorry, there was a problem we could not recover from.","text1":"The server returned a \"{{statusCode_statusText}}\"","help-url-text":"Help me resolve this"}};</script><!-- Sets the content of the translated text, IMPORTANT DO NOT REMOVE -->
+	<script>window.errorLocale={"en-US":{"language":"English US","header":"Sorry, there was a problem we could not recover from.","text1":"The server returned a \"{{statusCode_statusText}}\"","help-url-text":"Help me resolve this"},"en-GB":{"language":"English GB","header":"Sorry, there was a problem we could not recover from.","text1":"The server returned a \"{{statusCode_statusText}}\"","help-url-text":"Help me resolve this"}};</script><!-- Sets the content of the translated text, IMPORTANT DO NOT REMOVE -->
```
Comment (richard67) [inline templates/system/fatal-error.html]: Same as above: This change seems to be unrelated and not intentional since it is not mentioned in the description of this PR. Please revert it.

File: templates/system/build_incomplete.html (hunk 1 above)
Comment (Khanjan04) [reply templates/system/build_incomplete.html]: Yeah sure

File: templates/system/fatal-error.html (hunk 2 above)
Comment (Khanjan04) [reply templates/system/fatal-error.html]: Yeah sure

**Comment by Khanjan04** [comment]
Hey @richard67 I have made the changes please have a look now

**Comment by rdeutz** [comment]
Please add testing instructions. I makes testing possible for people who not having the ablity to read your mind or code.

**Comment by Khanjan04** [comment]
> Please add testing instructions. I makes testing possible for people who not having the ablity to read your mind or code.

Okay.

**Review by richard67 — APPROVED** [review]

**Comment by Khanjan04** [comment]
Thanks a lot @richard67 . When will you merge this PR? Just asking out of curiosity.

**Comment by richard67** [comment]
First it has to be tested with success by 2 testers who are not the author (so not you), then it will be set to status "RTC" (ready to commit), and then a maintainer will merge it sooner or later after a final review.

**Comment by Khanjan04** [comment]
Okay.

**Comment by richard67** [comment]
@Khanjan04 The description of this PR still doesn't include testing instructions. Please add them so people know how to test your PR. Thanks in advance.

**Comment by richard67** [comment]
@Khanjan04 And another question: Is there an issue here on GitHub which is fixed by this PR? If so, the link to the issue should be provided in the description of this PR by adding its number after the "#" in the "Pull Request for Issue # ." at the top of this PR.

**Comment by brianteeman** [comment]
@richard67 the issue is #32682

**Comment by richard67** [comment]
@brianteeman Thanks ... I guess I should have seen that too here when checking the "... mentioned this pull request 4 hours ago " message above here in the history.

**Comment by Quy** [comment]
Go to Content > Articles.
Click `New` button.
Click `Save` button.
`Title` and `Please fill in this field` should be in red, but they are not.

**Comment by drmenzelit** [comment]
This PR can't work, because it is removing the wrong scss code

@Khanjan04 can you please check again? And add the testing instructions, please.

**Comment by rdeutz** [comment]
closing, we asked three time to add testing instructions, that's enough time spend on this.

Can be reopend when all needed information are added

**Comment by Khanjan04** [comment]
> closing, we asked three time to add testing instructions, that's enough time spend on this.
> 
> Can be reopend when all needed information are added

Sorry I was busy with college work. I will do this as soon as possible
